src/Catalyst.Core.Modules.Cryptography.BulletProofs/NativeBinding.cs

```diff
@@ -70,9 +70,9 @@ internal static unsafe ISignature StdSign(byte[] privateKey, ReadOnlySpan<byte>
             {
                 fixed (byte* contextHandle = context)
                 {
-                    var error_code = std_sign(signature, publicKey, privateKey, message.Length > 0 ? messageHandle : empty, message.Length, context.Length > 0 ? contextHandle : empty, context.Length);
+                    var responseCode = std_sign(signature, publicKey, privateKey, message.Length > 0 ? messageHandle : empty, message.Length, context.Length > 0 ? contextHandle : empty, context.Length);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 202 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)